test/e2e/handler/static_addr_and_route_test.go

```diff
@@ -151,6 +169,35 @@ var _ = Describe("Static addresses and routes", func() {
 			})
 		})
 
+		Context("with bridge taking over the static networking of the slave", func() {
```

Comment:
You could maybe split the current Context into two contexts, the outer one would set up the static IP and route in BeforeEach and the inner one would  handle the policy with the capture.

Not a must though.

Reply:
Since the `AfterEach` is removing it all in one `updateDesiredStateAndWait(ipV4AddrAndRouteAndBridgeAbsent(firstSecondaryNic, bridgeName))` I prefer to do the settings in the same `BeforeEach`.

Reply:
Fair enough